packages/vite-node/src/cli.ts

```diff
@@ -93,5 +101,27 @@ async function run(options: CliOptions = {}) {
   for (const file of files)
     await runner.executeFile(file)
 
-  await server.close()
+  if (!options.watch)
+    await server.close()
+
+  event.on('updateModules', (modules: ModuleNode[]) => {
+    walkModules(modules, (module) => {
+      const { file, url } = module
+      const id = `/@fs/${slash(file!)}`
+      runner.moduleCache.delete(url)
+      runner.moduleCache.delete(id)
+      runner.executeId(id)
```

Comment:
* create
  just only `executeId`
* update
  delete the moduleCache and `executeId` to request vite transform module.
* delete
  delete module in moduleCache and request module. (TODO)